docs/insomnia/projects.md

```diff
@@ -17,12 +17,23 @@ There are three types of Projects: Default, Local, and Remote.
 _Projects are available via the main Insomnia dropdown. Here, you will find the Default Project listed first next to the home icon._
 
 ### Default Project
-This is a persistent Project that always exists for all users by default. It cannot be renamed or deleted. It appears next to the home icon in the Projects dropdown menu. 
+
+This is a persistent Project that always exists for all users by default. It cannot be renamed or deleted. It appears next to the home icon in the Projects dropdown menu.
 
 ### Local Project
+
 A local project can be created locally within the application to group Collections and Documents. This only exists on your client, it is not included in any export file, and is not part of your sync data.
 
 ### Remote Project
+
 If you are signed in and have access to a Team with Insomnia Sync enabled, that Team will be represented as a Remote Project on your client. When opening the Projects dropdown, Remote Projects that you have access to will automatically appear in the list.
 
-When creating a Request Collection inside a Remote Project, Insomnia will attempt to automatically enable sync. Collections that have been synced into a Remote Project can be pulled from the Dashboard after activating the Remote Project. For instance, inside the Insomnia Testing remote project, a user can pull Collections.
\ No newline at end of file
+When creating a Request Collection inside a Remote Project, Insomnia will attempt to automatically enable sync. Collections that have been synced into a Remote Project can be pulled from the Dashboard after activating the Remote Project. For instance, inside the Insomnia Testing remote project, a user can pull Collections.
+
+## Move Collections and Documents Between Projects
```

Comment:
There's a similar note about this here, in case it also needs updating: https://docs.insomnia.rest/insomnia/team-collaboration#share-request-collections-with-a-different-team

Reply:
Thank you. I created a task to revisit this page.